nio/responses.py

```diff
@@ -245,13 +246,15 @@ def from_dict(cls, parsed_dict):
         )
 
 
-@attr.s
-class Response(object):
-    uuid = ""          # type : str
-    start_time = None  # type : Optional[float]
-    end_time = None    # type : Optional[float]
-    timeout = 0        # type : int
-    transport_response = attr.ib(init=False, default=None)
+@dataclass
+class Response:
+    uuid: ClassVar[str] = ""
```

Comment:
Can't we convert those to be like the trabsport response? I think they were meant to be the same.

I'll check later with Weechat if something is broken.